app/Config/bootstrap.php

```diff
@@ -116,7 +116,7 @@
 /**
  * Sonerezh constants declarations
  */
-define('SONEREZH_VERSION', 'v1.2.5');
+define('SONEREZH_VERSION', 'v1.2.6');
```

Comment:
Don't change the version, it will be done when there's a new release.

But you could add your additions and changes in the CHANGELOG, under the Unreleased section (following the https://keepachangelog.com/en/1.1.0/ standard)